src/BridgeExtension.sol

```diff
@@ -21,7 +21,9 @@ contract BridgeExtension is IBridgeAndCall, IBridgeMessageReceiver, Initializabl
 
     PolygonZkEVMBridgeV2 public bridge;
 
-    constructor() {}
+    constructor() {
+        _disableInitializers();
```

Comment:
we could add the `bridge` address as immutable and we save ourselves the initialize function :D ( you can always import the initialize lib if you want to reserve the storage slot for a future tho)

Reply:
Good idea. Makes it a bit cleaner. I will do it.